CMakePresets.json

```diff
@@ -19,7 +19,7 @@
         "EXIV2_BUILD_UNIT_TESTS": true,
         "EXIV2_TEAM_WARNINGS_AS_ERRORS": true,
         "EXIV2_ENABLE_NLS": false,
-        "EXIV2_ENABLE_VIDEO": false
```

Comment:
From the discussion in https://github.com/Exiv2/exiv2/issues/2450 , it might make sense to keep this false.

Reply:
For the counterpoint, see https://github.com/Exiv2/exiv2/issues/2450#issuecomment-1369244447. My personal opinion is that there is a lot of benefit to be had in leaving it `true`

Reply:
My point of view is that we should enable it for most of the CI jobs (so that we can keep detecting issues with fuzzers, tests, etc) but disable it in the Release jobs for some time, until we are more confident about the quality of the code. I would also keep it disabled by default for the case in which somebody clones the project and try to compile the project with the default settings (without toggling CMake/Conan options).

Therefore, I would keep it `false` in the CMake base preset, and turn it ON into the CI jobs in which we are interested.

Reply:
Done here https://github.com/Exiv2/exiv2/pull/2448/commits/b6cf6fc92bb074aea8244805bcc0646cdc1241b6